docs/dev/remote-settings.rst

```diff
@@ -53,25 +47,9 @@ Use ``--dry-run`` to only print out the result of the synchronization.
 Client side
 -----------
 
-The Normandy client integration is available from Firefox 68 (`Bug 1506175
-<https://bugzilla.mozilla.org/show_bug.cgi?id=1506175>`_, `Bug 1538248
-<https://bugzilla.mozilla.org/show_bug.cgi?id=1538248>`_) and can be enabled
-by flipping this preference:
-
-- ``features.normandy-remote-settings.enabled``: ``true``
-
-In order to populate the list of recipes to be picked up by the Recipe Runner, trigger a manual synchronization in the Browser console:
-
-.. code-block:: javascript
-
-      ChromeUtils.import("resource://normandy/lib/RecipeRunner.jsm", this);
-
-      await RecipeRunner.loadRecipes();
-
-.. seealso::
-
-    * `Client setup with a local Remote Settings server
-      <https://remote-settings.readthedocs.io/en/latest/tutorial-local-server.html#prepare-the-client>`_
-
-The Normandy recipe runner will now be able to read the list of recipes to
-execute from the local Remote Settings database.
+The Normandy client integration was added in Firefox 66 (`Bug 1506175
+<https://bugzilla.mozilla.org/show_bug.cgi?id=1506175>`_), refined through
+Firefox 67 and 68 (`Bug 1538248
+<https://bugzilla.mozilla.org/show_bug.cgi?id=1538248>`_) and turned on by
+default in Firefox 68 (`bug 1513854
```

Comment:
Above you wrote 69

Reply:
Good catch, thanks! I thought it was 69 based on other discussions in the Normandy chat room, but @sciurus pointed out that it seems to have landed in 68.